lib_analysis.py

```diff
@@ -151,40 +151,61 @@ def compute_btag_weights(jets, mask_rows, mask_content, sf, jets_met_corrected,
 
 ############################################# HIGH LEVEL VARIABLES (DNN evaluation, ...) ############################################
 
-def evaluate_DNN(jets, good_jets, electrons, good_electrons, muons, good_muons, scalars, mask_events, nEvents, DNN, DNN_model, jets_met_corrected):
+def evaluate_DNN(jets, good_jets, electrons, good_electrons, muons, good_muons, scalars, mask_events, nEvents, DNN, DNN_model, jets_met_corrected, outdir="./"):
     
         # make inputs (defined in backend (not extremely nice))
         if jets_met_corrected:
-            jets_feats = ha.make_jets_inputs(jets, jets.offsets, 10, ["pt_nom","eta","phi","en","px","py","pz", "btagDeepB"], mask_events, good_jets)
+            jets_feats = ha.make_jets_inputs(jets, jets.offsets, 10, ["pt_nom","eta","phi","en","px","py","pz", "btagDeepFlavB"], mask_events, good_jets)
             met_feats = ha.make_met_inputs(scalars, nEvents, ["phi_nom","pt_nom","sumEt","px","py"], mask_events)
         else:
-            jets_feats = ha.make_jets_inputs(jets, jets.offsets, 10, ["pt","eta","phi","en","px","py","pz", "btagDeepB"], mask_events, good_jets)
+            jets_feats = ha.make_jets_inputs(jets, jets.offsets, 10, ["pt","eta","phi","en","px","py","pz", "btagDeepFlavB"], mask_events, good_jets)
             met_feats = ha.make_met_inputs(scalars, nEvents, ["phi","pt","sumEt","px","py"], mask_events)
         leps_feats = ha.make_leps_inputs(electrons, muons, nEvents, ["pt","eta","phi","en","px","py","pz"], mask_events, good_electrons, good_muons)
-
+        
+        if DNN == "save-arrays":
```

Comment:
das wird nicht funktionieren :D so ueberschreibst du dir die files, wenn du in verschiedenen batches prozessierst